src/applications/personalization/profile/tests/e2e/address-validation/addressValidation.js

```diff
@@ -0,0 +1,310 @@
+export const createAddressValidationResponse = type => {
+  if (type === 'valid-address') {
```

Comment:
One test I'd really like to see if when we get back a single, low confidence (`confidenceScore` < 90) address to confirm that we do show the validation screen in that situation.

Reply:
Added! 💪